app/lib/calendrical_events/models/calendrical_event.dart

```diff
@@ -15,6 +16,13 @@ import 'package:time/time.dart';
 import 'calendrical_event_types.dart';
 
 class CalendricalEvent {
+  /// The date and time when the event was created.
+  ///
+  /// Clients with a lower version than 1.7.9 will not have this field.
+  /// Therefore, we will always have events without a [createdOn] field in the
+  /// database.
+  final DateTime? createdOn;
```

Comment:
Technically, we don't need to parse the `createdOn` field because we don't need in the client. Should we still add it to the DTO and parse it?

Reply:
I'm fine with it 👍